Add JupyterLab instructions

juanfrh opened the pull request.

eleanor-broadway reviewed (changes requested):
All looks good apart from the step numbering. Are these supposed to be 1, 2, 3, etc.?

eleanor-broadway reviewed (approved):
Great, thanks!

juanfrh commented:
> All looks good apart from the step numbering. Are these supposed to be 1, 2, 3, etc.?

I've updated the step numbering. The rendered version showed 1, 2, 3... Nevertheless, I've updated the list to increase code readability. Thanks for the suggestion!
